lib/shelley/bench/Restore.hs

```diff
@@ -711,7 +769,7 @@ waitForNodeSync tr nw = loop 10
 
 data BenchmarkLog (n :: NetworkDiscriminant)
     = MsgNodeTipTick BlockHeader SyncProgress
-    | MsgRestorationTick SyncProgress
+    | MsgRestorationTick POSIXTime [SyncProgress]
```

Comment:
Why did I add this? Might not be needed _yet_.

Reply:
well, below you don't use it in toText, so probably superflouous ;-)

Reply:
But why not introduce `MsgRestorationTick [(POSIXTime,SyncProgress)]` to have this time for each sync progress? that would be useful probably